examples/air-quality-sensor-app/cc32xx/README.md

```diff
@@ -0,0 +1,232 @@
+# Matter `CC32XXSF` Lock Example Application
```

Comment:
This is very much not a lock.

I'm going to stop here.  Please check all the copy/paste to make sure it's actually correct.